lib/middleware.js

```diff
@@ -460,7 +460,7 @@ module.exports = function(srf, logger) {
       }).catch((err) => this.logger.info({err}, 'Error generating alert for parsing application'));
       logger.info({err}, `Error retrieving or parsing application: ${err?.message}`);
       res.send(480, {headers: {'X-Reason': err?.message || 'unknown'}});
-      app.requestor.close();
+      app.requestor.close(WS_CLOSE_CODES.GoingAway);
```

Comment:
@rammohan-y In this case we received a payload that is not a valid jambonz application.  That seems like it would map more correctly to a code of 1003
```
1003 indicates that an endpoint is terminating the connection
      because it has received a type of data it cannot accept
```
https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4.1